docs/reference/inference/elastic-infer-service.asciidoc

```diff
@@ -49,6 +48,10 @@ include::inference-shared.asciidoc[tag=chat-completion-docs]
 ==== {api-request-body-title}
 
 
+`chunking_settings`::
```

Comment:
Oops forgot to submit this feedback: At the moment there aren't any chunking options for EIS, so I think we can remove this.